test/core/extensions/coreIssuance.spec.ts

```diff
@@ -316,7 +330,242 @@ contract('CoreIssuance', accounts => {
     });
   });
 
-  describe('#redeem', async () => {
+  describe('#issue: RebalancingToken', async () => {
+    let subjectCaller: Address;
+    let subjectQuantityToIssue: BigNumber;
+    let subjectSetToIssue: Address;
+
+    let vanillaQuantityToIssue: BigNumber;
+    let vanillaSetToIssue: Address;
+
+    let initialShareRatio: BigNumber;
+    let rebalancingNaturalUnit: BigNumber;
+
+    const naturalUnit: BigNumber = ether(2);
+    let components: StandardTokenMockContract[] = [];
+    let componentUnits: BigNumber[];
+    let setToken: SetTokenContract;
+    let rebalancingToken: RebalancingSetTokenContract;
+
+    beforeEach(async () => {
+      components = await erc20Wrapper.deployTokensAsync(2, ownerAccount);
+      await erc20Wrapper.approveTransfersAsync(components, transferProxy.address);
+
+      const componentAddresses = _.map(components, token => token.address);
+      componentUnits = _.map(components, () => ether(4)); // Multiple of naturalUnit
+      setToken = await coreWrapper.createSetTokenAsync(
+        core,
+        setTokenFactory.address,
+        componentAddresses,
+        componentUnits,
+        naturalUnit,
+      );
+
+      initialShareRatio = DEFAULT_UNIT_SHARES;
+      rebalancingNaturalUnit = DEFAULT_REBALANCING_NATURAL_UNIT;
+      const initialSet = setToken.address;
+      const manager = managerAccount;
+      const proposalPeriod = ONE_DAY_IN_SECONDS;
+      const rebalanceInterval = ONE_DAY_IN_SECONDS;
+      const callData = Utils.bufferArrayToHex([
+        Utils.paddedBufferForPrimitive(manager),
+        Utils.paddedBufferForBigNumber(proposalPeriod),
+        Utils.paddedBufferForBigNumber(rebalanceInterval),
+      ]);
+      rebalancingToken = await coreWrapper.createRebalancingTokenAsync(
```

Comment:
for this `createRebalancingTokenAsync` method, make some of the parameters default to these constants so you dont need to keep defining them above